app/proto/search.proto

```diff
@@ -23,6 +24,11 @@ service Search {
     // Search for users in particular with extra filters, e.g. hosts
   }
 
+  rpc UserSearchV2(UserSearchReq) returns (UserSearchV2Res) {
+    // Search for users in particular with extra filters, e.g. hosts
+    // only returns used fields
+  }
+
```

Comment:
Is there supposed to be something in the brackets?

Reply:
No, empty. We have comments in there sometimes (like here) and if using JSON transcoding or other options, they go in there as well.

It's funky syntax!